test/helperTests.js

```diff
@@ -0,0 +1,54 @@
+var nohm = require(__dirname+'/../lib/nohm').Nohm;
+var args = require(__dirname+'/testArgs.js');
+var redis = args.redis;
+var helperLib = require(__dirname+'/../lib/helpers.js');
+var h = require(__dirname+'/helper.js');
+
+var UserLinkMockup = nohm.model('UserLinkMockup', {
```

Comment:
As far as I can see, the UserLinkMockup model can be deleted, because it is not used, right?

Reply:
Yep, copy/paste error. Removed.